package/src/components/PipecatAppBase.tsx

```diff
@@ -67,8 +74,16 @@ export interface PipecatBaseChildProps {
   handleDisconnect?: () => void | Promise<void>;
   /** Error message if connection fails */
   error?: string | null;
+  /** Response returned from starting the bot. */
+  rawStartBotResponse?: TransportConnectionParams | unknown;
+  /** Transformed start bot response. */
+  transformedStartBotResponse?: TransportConnectionParams | unknown;
```

Comment:
I believe my only question is why we need to store this. Are we going to display it somewhere ?

Reply:
Alright, I updated the code to allow the transformer to be an async function and store the transformed response in a state (because useMemo won't work with promises). This also makes sure that the transformer is only called once.